crt_portal/cts_forms/tests/integration-authed/report_detail.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from auth import login_as_superuser
+
+
+@pytest.mark.only_browser("chromium")
+def test_click_back_to_all(page):
+    """Opens report detail page and goes back to reports page"""
+
+    login_as_superuser(page)
+
+    page.goto("/form/view")
+
+    with page.expect_navigation():
+        page.evaluate("document.querySelector('.td-link').click()")
+
+    assert page.is_visible("#contact-info")
+
```

Comment:
This asserts that the correct kind of page has loaded, but I think this might still pass without your bug fix

It'd be cool to see this test:
1. Take note of what and how many reports are displayed on the list page.
2. Assert that the clicked report loads the correct ID
3. Assert that the next / prev is showing "1 of correct number"
4. Click the next arrow a few (for example, 3) times
5. Click the back arrow a few (for example, 2) times
6. Assert that the pagination is correct (for example, "2 of correct number"
7. Click back to all
8. Assert that we've gone back to the correct place.

Reply:
I did most of this but the test was timing out when I added all of it. I think I can use pytest-timeout to set a longer timeout for the test but I wasn't sure if that was a good idea.